CHANGELOG.md

```diff
@@ -7,15 +7,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 Try to keep listed changes to a concise bulleted list of simple explanations of changes. Aim for the amount of information needed so that readers can understand where they would look in the codebase to investigate the changes' implementation, or where they would look in the documentation to understand how to make use of the change in practice - better yet, link directly to the docs and provide detailed information there. Only elaborate if doing so is required to avoid breaking changes or experimental features from ruining someone's day.
 
 ## [Unreleased]
+
+## [1.6.1] - 2024-09-12
+
 ### Changed
-- Support context in assertions (#1907)
 
-## Added
+* Support context in assertions [#1907](https://github.com/openfga/openfga/pull/1907)
+
+### Added
 
 * Stack trace when logging panics [#1904](https://github.com/openfga/openfga/pull/1904)
 * Throttling metric `throttled_requests_count` for observing the number of throttled requests for a given throttling configuration [#1863](https://github.com/openfga/openfga/pull/1863)
 * New metric on number of allowed vs. non-allowed Check responses [#1911](https://github.com/openfga/openfga/pull/1911)
-* SQLite support (beta) [#1615](https://github.com/openfga/openfga/pull/1615)
+
+#### New datastore engine: SQLite (beta) [#1615](https://github.com/openfga/openfga/pull/1615)
```

Comment:
Per [KACL](https://keepachangelog.com/en/1.1.0/), this should be under the `Added` heading

Reply:
It's a subheading under the Added heading though

Reply:
Still not valid KACL. Wouldn't be an issue if we didn't state that we conform to KACL at the top of the changelog. Don't need the H4 anyway, it could just be the first item in the added section. 

I won't block this PR though.